frontend/server/src/DAO/UserRank.php

```diff
@@ -166,7 +166,7 @@ public static function getAuthorsRank(
     }
 
     /**
-     * @return array{ranking: list<array{author_ranking: int, username: string, classname: string}>, total: int}
+     * @return array{ranking: list<array{author_ranking: int|null, author_score: float, country_id: null|string, username: string, name: null|string, classname: string}>, total: int}
```

Comment:
```suggestion
     * @return AuthorsRank
```

_parece_ que se debe poder reutilizar el tipo!